samples/meetings-sidepanel/csharp/SidePanel/ClientApp/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Sample app which demonstrate use of live share sdk inside meeting sidepanel."
```

Comment:
Sample app which demonstrates how to use of live share SDK inside meeting side panel.